text/0018-publish-update.md

```diff
@@ -0,0 +1,180 @@
+- Feature Name: publish and update
+- Start Date: 2018-10-24
+- RFC PR: [#27](https://github.com/qri-io/rfcs/pull/27)
+- Issue: <!-- (leave this empty) -->
+
+# Summary
+[summary]: #summary
+
+Add "publish" and "update" as patterns for broadcasting when a dataset is ready for public consumption, and an update pattern for generating & syncing version histories.
+
+# Motivation
+[motivation]: #motivation
+
+Right now Qri is a little _too_ "public". Any time I add anything to Qri and go online, I have no tools for saying "I don't want people downloading this dataset", or "this dataset isn't ready for public consumption". This is creating "use paralysis", where I don't feel as comfortable as I should "practicing" on Qri, because any and everything I save is visible to the world. If I make a mistake, I have no clear way of telling another user "hey don't grab these updates!".
+
+Speaking of updates, I don't have any clear mechanism for re-fetching datasets that's been updated since I "added" it. We need a quick and easy way to syncronize my remote copy of a dataset with the lastest version after it's been published.
+
+# Guide-level explanation
+[guide-level-explanation]: #guide-level-explanation
+
+This RFC introduces two new concepts to Qri that will come with new commands and, will modify some existing behaviors. Publish and Update will work together to allow users to explicitly opt-in to sync points
+
+Because this process will be run frequently, I'd like to make the common use patterns as simple as possible, to the point that `qri save` should come with a `publish` flag that combines saving an update & publishing it into one step.
```

Comment:
First time, I misread the end of this sentence as "saving and updating & publishing it..." which reminds me that "update" is still an overloaded term. An "update" is also how we refer to new snapshots applied with the `save` command, as opposed to a "patch". This could lead to confusion, or perhaps you meant the similarity of meanings to be on purpose? In that case the `qri update` command will both retrieve and create new "updates".

Reply:
You're right, the term is overloaded, and I didn't mean for previous RFC's version of `update` to collide with this one. 

I now think the former RFC's use of the phrase "update" was a mistake, and instead should be called something that refers to dataset _completeness_, instead of referring to _temporality_. I'm working on an RFC that'll hopefully clarify terms around the "dataset completness" question. I'll cite that poor use of the word `update` as a need for such a document.